earth2mip/grid.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+import numpy as np
+import dataclasses
+from earth2mip import schema
+
+
+@dataclasses.dataclass(frozen=True)
+class LatLonGrid:
+
+    lat: List[float]
+    lon: List[float]
+
+    @property
+    def shape(self):
+        return (len(self.lat), len(self.lon))
+
+
+def regular_lat_lon_grid(
```

Comment:
Nit: if this is regular lat lon grid, can we just call it `lat_lon_grid` and anything different is then `something_lat_lon_grid`

Seems a little too verbose

Reply:
rename to equiangular_lat_lon?

Reply:
done.